com.unity.render-pipelines.universal/Runtime/ScriptableRenderer.cs

```diff
@@ -797,6 +797,10 @@ protected static ClearFlag GetCameraClearFlag(ref CameraData cameraData)
             if (debugHandler != null && debugHandler.IsActiveForCamera(ref cameraData) && debugHandler.IsScreenClearNeeded)
                 return ClearFlag.All;
 
+            // XR: TODO, remove once we have visible area of occlusion mesh available
```

Comment:
One last minor change, can you use `// XRTODO` here ? we used that for most of SRP.

Reply:
Sure